i18n/es/docusaurus-theme-classic/navbar.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": {
+    "message": "LunarVim",
+    "description": "The title in the navbar"
+  },
+  "item.label.Docs": {
+    "message": "Documentacion",
```

Comment:
Is there a problem if we put accent marks here?
In this case it should be "Documentación"

Reply:
No problem with that, I thought that navbar and the font would have some issues with UTF-8, but I guess it's fine

Reply:
Let's hope it doesn't cause problems haha.
Nice PR